### DISCUSSION TITLE
I trying to submit a project to my school but grade said: "verify that you have enabled Github Pages in the setting for your wdd131 named repository" how I do a look to verify this issue?

### DISCUSSION THREAD

**Discussion by Reypao** [community/community#143175] [GitHub Education]
### Select Topic Area

Question

### GitHub Feature Area

Actions

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Reypao! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them. Note, however, that I did remove the file you uploaded to this post. Can you copy/paste what's needed in the post so that someone can provide help with your question?

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by joegeorge022** [comment]
It might be because the project isn't deployed on **GitHub Pages**. Follow the steps below to enable deployment:

---

### Steps to Deploy

1. **Open Repository Settings**  
   Go to the settings for your repository (`wdd131`).

2. **Navigate to Pages**  
   In the left sidebar, find the **"Code and automation"** section and click on **"Pages"**.

3. **Select Deployment Source**  
   Set the **Source** to "Deploy from a branch" (this might already be enabled).

4. **Choose Branch**  
   From the dropdown, select the branch as **"main"**.

5. **Save Changes**  
   Click on **"Save"** to finalize the settings.

---

Once you've completed these steps, your project should be deployed successfully.

> **Note**: Now, try submitting your project again.

### Let me know if you need further help 🙂✌️
